qa-heal.planx-pla.net/metadata/aggregate_config.json

```diff
@@ -23,9 +23,6 @@
                 "type": "array",
                 "default": []
             },
-            "data_dictionary": {
```

Comment:
this field is causing issues for the new AggMDS schema feature, the error was `bytes can be at most 32766 in length`

one of the JCOIN metadata has a super long pure text data dict https://jcoin.datacommons.io/mds/metadata/1U2CDA050098-01_b. This is causing the issue. I think we should use text typle for these long fields, so maybe we should add another field type in the new AggMDS @craigrbarnes @tianj7